sdk/notificationhubs/notification-hubs/src/client/beginSubmitNotificationHubJob.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AbortController, AbortSignalLike } from "@azure/abort-controller";
+import { CancelOnProgress, OperationState, SimplePollerLike } from "@azure/core-lro";
+import { NotificationHubJob } from "../models/notificationHubJob.js";
+import { NotificationHubsClientContext } from "./index.js";
+import { OperationOptions } from "@azure/core-client";
+import { getNotificationHubJob } from "./getNotificationHubJob.js";
+import { submitNotificationHubJob } from "./submitNotificationHubJob.js";
+
+/**
+ * Submits a Notification Hub job and creates a poller to poll for results.
+ * @param context - The Notification Hubs client.
+ * @param notificationHubJob - The Notification Hub import/export job to start.
+ * @param options - The operation options.
+ * @returns A poller which can be called to poll until completion of the job.
+ */
+export async function beginSubmitNotificationHubJob(
+  context: NotificationHubsClientContext,
+  notificationHubJob: NotificationHubJob,
+  options: OperationOptions = {}
+): Promise<SimplePollerLike<OperationState<NotificationHubJob>, NotificationHubJob>> {
+  let submittedJob = await submitNotificationHubJob(context, notificationHubJob, options);
+
+  type Handler = (state: OperationState<NotificationHubJob>) => void;
```

Comment:
This function implementation duplicates and adopts the implementation of https://github.com/Azure/azure-sdk-for-js/blob/074b7d557f81d22fc89abaf271e7b59c18416a0e/sdk/core/core-lro/src/poller/poller.ts#L46
`buildCreatePoller` is an internal function in `core-lro` but there is a question here about exporting it to be used here instead. That is an implementation detail and can be discussed separately outside this PR.

Reply:
I think we could have `core-polling` which is separate from `core-lro` since it is just a subset of LRO operations.